pages/docs/concepts/consumer.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Consumer
+weight: 4
+---
+
+## What is a Consumer?
+In an Event Driven Architecture (EDA), a consumer is a component of an application or messaging system that listens for a particular event from a broker and takes the necessary action in response.
```

Comment:
```suggestion
In an Event Driven Architecture (EDA), a consumer is an application that listens for a particular event from a broker and reacts to it.
```

> application

I think it is good to be consistent with producer document that was just released

> takes the necessary action in response

I would avoid `response` as it is known term in request/response pattern, and might be suggest consumer sends a response